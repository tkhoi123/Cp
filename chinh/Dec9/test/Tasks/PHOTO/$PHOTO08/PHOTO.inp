27
5 1 20 24 13 6 27 11 10 14 21 4 23 3 25 9 2 8 22 26 17 19 18 12 7 15 16 
14 18 5 13 19 12 21 4 3 11 25 1 2 9 8 24 23 10 27 6 22 26 17 7 15 16 20 
18 5 24 13 27 9 19 11 12 10 14 21 4 23 3 25 1 2 8 6 22 26 17 7 15 16 20 
18 5 16 24 13 27 19 26 11 12 10 14 21 23 25 1 9 2 3 6 4 22 17 7 20 15 8 
18 5 24 27 19 11 12 10 13 14 21 25 4 23 3 1 9 2 8 6 22 26 17 7 15 16 20 
