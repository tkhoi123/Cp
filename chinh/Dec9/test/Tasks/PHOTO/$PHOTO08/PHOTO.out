18 5 24 13 27 19 11 12 10 14 21 4 23 3 25 1 9 2 8 6 22 26 17 7 15 16 20 